### PR TITLE
id23_FrequencyOfCharactersInSortedManner

### PR DESCRIPTION
**Pull request by ihccik**
cheet updated
file name updated
current solution checked
LinkedHashMap solution added
Solution with string functions added

### CODE REVIEW

File: src/loops_character/id23_FrequencyOfCharactersInSortedManner.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import java.util.TreeMap;
+
+public class id23_FrequencyOfCharactersInSortedManner {
+
+    /*
+        Write a function that takes a String as input and returns the frequency of each character sorted by character.
+        Assume that all characters in the input string are capitalized. And input string always consist of only letters.
+        Input string can not contain space character.
+        If input string is null just return null, is it is empty just return empty string.
+        Input String can contain space, and space is evaluated a
```
Comment (Cemil5) [inline src/loops_character/id23_FrequencyOfCharactersInSortedManner.java:15]: line 13 says string cannot contain space character but line 15 says can contain

**Review by Cemil5 — CHANGES_REQUESTED** [review]
line 13 says string cannot contain space character but line 15 says can contain. I think we should delete line 15

File: src/loops_character/id23_FrequencyOfCharactersInSortedManner.java (hunk 1 above)
Comment (ihccik) [reply src/loops_character/id23_FrequencyOfCharactersInSortedManner.java:15]: 😊

**Comment by ihccik** [comment]
@Cemil5 done. Thanks for detailed review.

**Review by Cemil5 — APPROVED** [review]

**Pull request merged**